var/spack/repos/builtin/packages/crtm-fix/package.py

```diff
@@ -5,31 +5,34 @@
 
 import os
 
-from spack import *
+from spack.package import *
 
 
 class CrtmFix(Package):
-    """CRTM coeffecient files"""
+    """CRTM coefficient files"""
 
     homepage = "https://github.com/NOAA-EMC/crtm"
-    url = "ftp://ftp.ucar.edu/pub/cpaess/bjohns/fix_REL-2.3.0_emc.tgz"
+    url = "ftp://ftp.ssec.wisc.edu/pub/s4/CRTM/fix_REL-2.3.0_emc.tgz"
 
-    maintainers = ["kgerheiser"]
+    maintainers = [
+        "BenjaminTJohnson",
+        "edwardhartnett",
+        "AlexanderRichert-NOAA",
+        "Hang-Lei-NOAA",
+        "climbfuji",
+    ]
 
-    version("2.4.0", sha256="7924285b8aa25b0b864643f03e7f281ca1816958e24c76aa9531b3ca902bf6e9")
     version("2.4.0_emc", sha256="88d659ae5bc4434f7fafa232ff65b4c48442d2d1a25f8fc96078094fa572ac1a")
-    version("2.3.0_emc", sha256="fde73bb41c3c00666ab0eb8c40895e02d36fa8d7b0896c276375214a1ddaab8f")
+    version("2.3.0_emc", sha256="1452af2d1d11d57ef3c57b6b861646541e7042a9b0f3c230f9a82854d7e90924")
```

Comment:
Why is this commit hash changing? Did the tag move in the crtm repo or is it correcting a mistake?